exalearn-api/ExaLearn.Tests/Services/UserServiceTests.cs

```diff
@@ -120,5 +152,35 @@ public async Task GetHrUserHistoryByIdAsync_HrHistroryModelIsValid_ResultWithVal
             Assert.Equal("Aaron Ramsdale", result[0].FullName);
             Assert.Equal("Sam Johnstone", result[1].FullName);
         }
+
+        [Fact]
+        public async Task GetHrAssignedTestByIdAsync_HrAssignTestModelIsValid_ResultWithValidLevel()
+        {
+            // Arrange
+            var test = _mockAssignTestRepository.Object.GetByIdAsync(1);
+
+            // Act
+            var result = await _userService.GetHrAssignedTestByIdAsync(test.Id);
+
+            // Assert
+            Assert.NotNull(result);
+            Assert.Equal(LevelType.Beginner, result[0].Level);
+            Assert.Equal(LevelType.Intermediate, result[1].Level);
+        }
+
+        [Fact]
+        public async Task GetHrAssignedTestByIdAsync_HrAssignTestModelIsValid_ResultWithAssignerName()
+        {
+            // Arrange
+            var test = _mockAssignTestRepository.Object.GetByIdAsync(1);
+
+            // Act
+            var result = await _userService.GetHrAssignedTestByIdAsync(test.Id);
+
+            // Assert
+            Assert.NotNull(result);
+            Assert.Equal("Bob Holland", result[0].AssignedBy);
+            Assert.Equal("Joe Howard", result[1].AssignedBy);
+        }
```

Comment:
make two tests to one test
Make Assert region: 
Assert.NotNull(result);
Assert.Equal(LevelType.Beginner, result[0].Level);
Assert.Equal("Bob Holland", result[0].AssignedBy);
Assert.Equal(LevelType.Intermediate, result[1].Level);
Assert.Equal("Joe Howard", result[1].AssignedBy);

Reply:
Done